main/templates/quickstart/deploy.md

```diff
@@ -1,11 +1,19 @@
-To deploy your app to App Engine, you will need to register a project to create your project ID, which will determine the URL for the app.
+Deploy your app to Google App Engine, so that anyone can access your app in the cloud. You will need to register a project (or re-use an existing one) and obtain the Project ID, as this will determine the web address for the app.
 
 1. In the Cloud Platform Console, go to the [Projects page](https://console.cloud.google.com/project) and select or create a new project.
-2. Note the project ID that you created above.
-3. Upload your application to Google App Engine by invoking the following command from the root directory. This opens a browser window for you to sign in using your Google account. You'll be providing the project ID as the argument for `--project`.
+
+2. Note the Project ID of that project.
+
+3. Start a console and change into the `hello` sample app directory:
+
+    ```
+    cd hello
+    ```
+
```

Comment:
if they are following the whole thing at once.. it's not really needed to mention again to enter the directory.. I would kill that step..